hangman/lib/hangman/game.ex

```diff
@@ -1,80 +1,34 @@
 defmodule Hangman.Game do
```

Comment:
# -15

There are several issues here.

First, although you start a server called Hangman.State, you never actually use it. Instead, you just pass around a structure defined in the same module.

Second, the idea is that the game module itself should not change. It already contains all the implementation of the game logic—you just needed to wrap it in a server. We've done this in class—we have a server that maintains state (the thing returned by new_game) and that passes this thing to make_move and tally.